src/js/express.js

```diff
@@ -0,0 +1,148 @@
+// Express instance represented as a component
+//
+// Add additional routers and middleware as child components of this module
+"use strict";
+var fluid       = fluid || require("infusion");
+var namespace   = "gpii.express";
+var express     = fluid.registerNamespace(namespace);
+var path        = require("path");
+var exphbs      = require("express-handlebars");
+var handlebars  = require("handlebars");
+
+express.initExpress = function(that) {
+    if (!that.options.config || !that.options.config.express) {
+        console.error("Cannot initialize express because you have not supplied a 'config' option.");
+        return;
+    }
+
+    // We need these to be model variables so that other grades can listen and take action when they are changed.
+    var express  = require("express");
+    that.applier.change("express", express());
+    that.applier.change("router", express.Router());
+    that.model.express.use(that.options.path, that.model.router);
+
+    var helpersToLoad = [];
+    if (that.options.components) {
```

Comment:
I am working on the distributeOptions refactor at the moment.  I could use some clarification on:

> All this polymorphic code is wrapped up in a function and is not accessible for people extending the system to customise.

I assume you are talking about the initial creation of express?  I need more details on that, and also on what a good alternative approach would be.